CMakeLists.txt

```diff
@@ -155,6 +155,7 @@ add_definitions(-DLLVMREV=${LLVMREV})
 
 include_directories(${LLVM_INCLUDE_DIRS})
 include_directories(from_cpython/Include)
+include_directories(from_cpython/Modules)
```

Comment:
Hmm what do you think about doing `#include "../Modules/socketmodule.h"` instead of adding Modules/ to the search path?  I know that it's ugly but CPython does something similar as well, and it might be risky to add Modules/ to the search path if it wasn't intended to be on it.